include/spirv/unified1/spirv.core.grammar.json

```diff
@@ -4102,6 +4102,46 @@
       "capabilities" : [ "Addresses", "VariablePointers", "VariablePointersStorageBuffer" ],
       "version" : "1.4"
     },
+    {
+      "opname" : "OpColorAttachmentReadEXT",
+      "class"  : "Image",
+      "opcode" : 4160,
+      "operands" : [
+        { "kind" : "IdResultType" },
+        { "kind" : "IdResult" },
+        { "kind" : "IdRef", "name" : "'Attachment'" },
+        { "kind" : "IdRef", "name" : "'Sample'", "quantifier" : "?" }
+      ],
+      "extensions" : [ "SPV_EXT_shader_tile_image" ],
+      "capabilities" : [ "TileImageColorReadAccessEXT" ],
+      "version" : "1.5"
```

Comment:
This version is only for core spir-v versions. Please remove from all instructions.

Reply:
Sorry I think I messed this up. I believe this should be:
```
"version" : None
```

I believe no entry is interpreted as 1.0.

Reply:
Aha. No worries. Added version fields back - with 'None'.

Reply:
This was updated in the most recent commit.